packages/router/package.json

```diff
@@ -22,9 +23,10 @@
     "react-dom": "^16.13.1"
   },
   "scripts": {
-    "build": "yarn build:js",
+    "build": "yarn build:js && yarn build:types",
```

Comment:
@Tobbe this was missing from the router typescript setup

Reply:
ttsc invoked from somewhere else must be building the types for the router as well, because I get the types even without your change here when I run `yarn build` in redwood root. And adding that code doesn't seem to change what is output.

It is needed if you try to go in to /packages/router and manually build just there. 

Thanks for the heads up, I'll write a PR

Reply:
https://github.com/redwoodjs/redwood/pull/1722